app/ErrorHandler.scala

```diff
@@ -14,8 +14,9 @@
  * limitations under the License.
  */
 
+import play.Logger
```

Comment:
And is it better to replace this with the Logging trait? 
`import play.api.Logging`

then use Logging trait on the class (with Logging) to use the logger variable

so we wouldn't have to explicitly create `val logger = Logger.of(appName)` unless there is a reason we specifically need to?

Reply:
Probably. I wasn't aware of that trait